chat-plugins/lottery.js

```diff
@@ -91,7 +91,7 @@ exports.commands = {
                 this.privateModCommand("(" + Tools.escapeHTML(user.name) + " has ended the game of lottery.)");
                 break;
             case 'setlimit':
-                if (!this.can('hotpatch')) return this.errorReply("Access denied.");
+                if (!this.can('hotpatch', null, room)) return false;
```

Comment:
``hotpatch`` is not a room command, I think ``if (!this.can('hotpatch')) return false;`` would be good here.